nengo/synapses.py

```diff
@@ -72,8 +73,13 @@ def general_step(signal, output, x, y, num, den):
         y.appendleft(np.array(output))
 
     def make_step(self, dt, output, method='zoh'):
-        num, den, _ = cont2discrete((self.num, self.den), dt, method=method)
-        num = num.flatten()
+        num, den = self.num, self.den
+        if self.analog:
+            num, den, _ = cont2discrete((num, den), dt, method=method)
+            num = num.flatten()
+
+        if den[0] != 1.:
+            raise ValueError("First element of the denominator must be 1")
```

Comment:
[`scipy.lifter`](http://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.lfilter.html) handles this by normalizing `num` and `den` by `den[0]` if it isn't 1. Should we adopt that instead? Personally, I spent a buttload of time trying to debug my filter because of this issue, so I'm in favour of the `ValueError`, just wanted to raise it as another possibility.